templates/web/base/report/_item_small.html

```diff
@@ -30,3 +30,19 @@
     [% ELSIF problem.bodies_str_ids.size == 0 %] [% loc('(not sent to council)') %]
     [% END %]
 [% END %]
+<div class='assigned-to' style='margin-top:0.25em'>
+[%  IF c.user.has_permission_to('report_inspect', c.cobrand.body.id ) %] 
+    [% IF problem.shortlisted_user %] [%# this report is on a shortlist %]
+        <span class="inline-text assigned-to-label" for="assignment">[% loc('Assigned to:') %] </span>
+        <strong>
+        [% IF problem.shortlisted_user.name %]
+            [% problem.shortlisted_user.name %]
+        [% ELSE %]
+            [% problem.shortlisted_user.username %]
+        [% END %]
+        </strong>
+    [% ELSE %]
+        <strong>(unassigned)</strong>
+    [% END %]
+[% END %]
```

Comment:
this is the same as `report-assignment.html`, I think?

Reply:
Yep, there were some slight differences in the presentation that were at odds with each other but I've resolved that and created a new mini-template.